src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -276,12 +276,20 @@ def acr_login(cmd,
             password=password,
             resource_group_name=resource_group_name)
 
-        logger.warning("You can perform manual login using the provided access token below, "
-                       "for example: 'docker login loginServer -u %s -p accessToken'", EMPTY_GUID)
+        logger.warning("Note: The token in both the accessToken and refreshToken fields is "\
+        "an ACR Refresh Token, not an ACR Access Token. This ACR Refresh Token cannot be used "\
+        "directly to authenticate with registry APIs such as pushing/pulling images and listing "\
+        "repositories/tags. This ACR Refresh Token must be subsequently exchanged for an ACR Access. "\
+        "Please see https://aka.ms/acr/auth/oauth")
+
+        logger.warning("You can perform manual login using the provided refresh token below, "
```

Comment:
Just realized that we need to change the help message to use the term "refresh token" as well
![image](https://github.com/user-attachments/assets/64993c6f-193f-45c3-866a-790651170332)